registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -97,6 +97,7 @@
     "sha256:d07762fd9a7e9f3e2cffa229c222c5d75d2b9185494dbd6a2962cb51dcafa370": ["v1.11.0-beta.1"]
     "sha256:11a86e5056384f15b544ecd77f551ce9faabb06800b34513007da5d2b75cc085": ["v1.11.0-beta.2"]
     "sha256:d115f04597212b26bf2f17e9b056821f86a9cc5ad5bd4dcb7bd27c5586df7f35": ["v1.11.0-rc.0"]
+    "sha256:e2d630371969d670bcf96b636f030bf07ebad61f8f06a93f2fe01dab6eee478f": ["v1.11.0-rc.1"]
```

Comment:
SHA `e2d630371969` matches cluster-api-controller v1.11.0-rc.1 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:e2d630371969d670bcf96b636f030bf07ebad61f8f06a93f2fe01dab6eee478f/details?inv=1&invt=Ab5TGA).